assemblies/cde-renderer/feature/pom.xml

```diff
@@ -21,5 +24,32 @@
       <classifier>features</classifier>
       <type>xml</type>
     </dependency>
+
+    <dependency>
+      <groupId>org.pentaho.ctools</groupId>
+      <artifactId>cde-core-impl</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <!-- Transitive dependency of cde-core-impl/Commons JXPath-->
+    <dependency>
+      <groupId>commons-beanutils</groupId>
+      <artifactId>commons-beanutils</artifactId>
+      <version>${commons-beanutils.version}</version>
+    </dependency>
```

Comment:
According to the guidelines this should only have groupId and artifactId, the rest should be in the root module.
(https://github.com/pentaho/pentaho-coding-standards/wiki/1.3.-Managing-Dependencies)